addons/mail/static/src/core/web/@types/models.d.ts

```diff
@@ -1,15 +1,36 @@
 declare module "models" {
-    export interface Discuss  {
-        inbox: Thread,
-        stared: Thread,
-        history: Thread,
+    export interface Activity {
+        dateCreateFormatted: Readonly<string>;
+        dateDeadlineFormatted: Readonly<string>;
+        dateDoneFormatted: Readonly<string>;
+        edit: () => Promise<object>;
+        markAsDone: (attachmentIds: number[]) => Promise<void>;
+        markAsDoneAndScheduleNext: () => Promise<unknown>;
+        remove: ({ broadcast: boolean }) => void;
+    }
+    export interface Message {
+        canForward: (thread: Thread) => boolean;
+        canReplyAll: (thread: Thread) => boolean;
     }
     export interface Store {
-        activityCounter: number,
-        activity_counter_bus_id: number,
-        activityGroups: Object[],
+        _onActivityBroadcastChannelMessage: ({ data: unknown }) => void;
```

Comment:
we should typedef the broadcast channel message we send ({ type: ..., payload: object (with fields of activity model)})